tests/unit/amo/components/TestScreenShots.js

```diff
@@ -4,22 +4,27 @@ import { renderIntoDocument } from 'react-dom/test-utils';
 import { PhotoSwipeGallery } from 'react-photoswipe';
 
 import ScreenShots, {
-  HEIGHT,
-  WIDTH,
   thumbnailContent,
 } from 'amo/components/ScreenShots';
 
+const HEIGHT = 400;
+const WIDTH = 200;
+
 describe('<ScreenShots />', () => {
```

Comment:
Could you please update this description with `__filename`?

Reply:
Also, I see that we are using `renderIntoDocument`. It could be nice to rewrite the test case with Enzyme.

Reply:
ahh..ok sure!